inc/class_datacache.php

```diff
@@ -687,15 +684,14 @@ function update_statistics()
 		');
 		$topposter = $db->fetch_array($query);
 
-		$query = $db->simple_select('users', 'COUNT(*) AS posters', 'postnum>0');
+		$query = $db->simple_select('users', 'COUNT(uid) AS posters', 'postnum>0');
 		$posters = $db->fetch_field($query, 'posters');
 
 		$statistics = array(
 			'time' => TIME_NOW,
-			'top_forum' => (array)$forum,
 			'top_referrer' => (array)$topreferrer,
 			'top_poster' => (array)$topposter,
-			'posters' => (int)$posters,
```

Comment:
> Does this means I'm entitled to revert your changes on my next PRs?

If that will be reasonable, feel free to. It's not like my changes should stay forever. But as @JN-Jones said, there is no point of casting an integer to integer. Actually, I see I used 2 redundant `(int)` castings too for some reason and will remove them in the next commit.

Also, I don't remember who, when and why made every PR... Do you?